packages/experiments/src/components/TilesList/TilesList.tsx

```diff
@@ -122,11 +125,37 @@ export class TilesList<TItem> extends React.Component<ITilesListProps<TItem>, IT
           role={role ? 'presentation' : undefined}
           getPageSpecification={this._getPageSpecification}
           onRenderPage={this._onRenderPage}
+          ref={this.listRef}
+          usePageCache={true}
+          {...this.props.listProps}
         />
       </FocusZone>
     );
   }
 
+  public scrollToIndex(index: number, mode: ScrollToMode = ScrollToMode.auto): void {
+    if (this.listRef.current) {
+      this.listRef.current.scrollToIndex(
+        index,
+        (itemIndex: number) => {
+          let cellHeight = 0;
+          if (this.props.getItemHeight) {
```

Comment:
Shouldn't `TilesList` itself be able to calculate the offset from its grid data?

Reply:
yes it can :)  so, I have removed the getItemHeight callback in the latest PR iteration.  Thank you